pipelines/ApTemplate.yaml

```diff
@@ -0,0 +1,12 @@
+description: Alert Production template building pipeline specialized for HiTS-2015
+# This pipeline assumes the working repo has raws, calibs, refcats, and a skymap,
+# and that you have already run the RunIsrForCrosstalkSources.yaml pipeline.
+
+imports:
+  - location: $AP_PIPE_DIR/pipelines/DarkEnergyCamera/ApTemplate.yaml
+parameters:
+  coaddName: deep
```

Comment:
Would it be better to transition to "goodSeeing" instead of "deep"?

Reply:
Not right now. For one thing, we'd have to regenerate the templates for Gen 2 as well.

Maybe once we finish getting rid of all the Gen 2 stuff we can revisit the dataset contents; for now the goal is to reproduce an exact Gen 3 equivalent of what's already here.